### DISCUSSION TITLE
Blog on squashing commits in git

### DISCUSSION THREAD

**Discussion by avinashupadhya99** [EddieHubCommunity/support#1239] [Show and tell]
Hey folks, I recently spent a couple of hours writing a blog on squashing commits using GitKraken and using the command line as well. Any feedback/review is much appreciated. I am hoping to write more blogs in the near future. Here's a link to the story - https://avinashupadhya99.medium.com/combine-old-git-commits-using-gitkraken-59b6f5cf4994

I can also provide an author's link if anybody has exhausted their free reading and needs one. I have found that opening it in an incognito window works well for this.

**Comment by BOLT04** [comment]
Great article @avinashupadhya99, it's concise, gets to the point and has images illustrating the steps 👍. Here is some feedback:
- I think you can add a small heading (like an `h2`) before this paragraph:

> Let’s do the same thing we did with the GUI. We will be squashing the second and the third commit using command line. Just like with the GUI, you will need to make sure you are on a branch which has HEAD on it.

   The heading would add a clear separation between using the GUI and using the command line.

- Also, you used [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for the commit messages I believe. It would be cool if you referenced that at the end (for the curious people that don't know about it 😃)

**Reply by avinashupadhya99** [reply]
Thank you for the feedback! I will definitely incorporate these 😃
